src/sinks/aws_s_s/config.rs

```diff
@@ -87,7 +87,7 @@ pub enum Encoding {
     Json,
 }
 
-pub fn message_group_id(
+pub(super) fn message_group_id(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏This is a good start but, there are still a bunch of raw `pub` structs, functions, traits, in the changeset that should be `pub(super)`.